README.md

```diff
@@ -500,6 +500,7 @@ Table of Contents
   * [LoginRadius](https://www.loginradius.com/) — Managed User Authentication service for free. Email registration and 3 social providers.
   * [logintc.com](https://www.logintc.com/) — Two-factor authentication (2FA) by push notifications, free for 10 users, VPN, Websites and SSH
   * [meterian.io](https://www.meterian.io/) - Monitor Java, Javascript, .NET, Scala, Ruby and NodeJS projects for security vulnerabilities in dependencies. Free for one private project, unlimited projects for open source.
+  * [MojoAuth](https://mojoauth.com/) - MojoAuth makes it easy to implement Passwordless on your web, mobile or any application in minutes.
```

Comment:
`easy to implement Passwordless authentication...` i guess reads better. 

I don't see any indication that this will remain free after Beta, which is required for being on the list

Reply:
We will soon be releasing pricing page and the basic features will be free forever. We will share the info with you soon.

Reply:
The pricing page is up now. 
https://mojoauth.com/pricing